README.md

```diff
@@ -1 +1,205 @@
-# metrics
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🚀 Live Presentation](#live-presentation)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Available Scripts](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metrics Webapp <a name="about-project"></a>
+
+> 🚧 "Digital Currency" is about working with the real live data from the Blockchain API. It's a web application for a company that provides informations such as trending coins, coins' price,and crypto transaction. The application allow users to get full information related to the crypto. 🚧
+
+## 🛠 Built With <a name="built-with">REACT-REDUX</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JAVASCRIPT</a></li>
+    <li><a href="https://getbootstrap.com/docs/5.3/getting-started/introduction/">BOOTSTRAP</a></li>
+    <li><a href="https://webpack.js.org/guides/getting-started/">WEBPACK</a></li>
+    <li><a href="https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow">GITFLOW</a></li>
+    <li><a href="https://reactrouter.com/en/main/start/tutorial#setup">REACT ROUTER DOM V6</a></li>
+    <li><a href="https://react.dev/learn">REACTJS</a></li>
+    <li><a href="https://redux-toolkit.js.org/">REDUX TOOLKIT</a></li>
+    <li><a href="https://jestjs.io/docs/getting-started">JEST</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Key Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Responsive design
+- Clear and intuitive metrics webapp user interface.
+- Redux for state management.
+- API interaction via Redux and use the data to display coins and coin details data in a UI.
+- Use React Router to navigate between pages in the app.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+
+## 🔭 Live Demo <a name="future-features"></a>
+
+[Live demo](https://strong-travesseiro-0c6db9.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Presentation of the project -->
+
+## 🔭 Video Presentation <a name="future-features"></a>
+
+Coming Soon
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#project-documentation), Please record a video presenting the features of the project you built and add the video linked here in the readme file. It is a video with a maximum length of 5 minutes. The content of the video should include:
   - a description of the project.
   - a demo of the project features.
   - you should also highlight some interesting piece of code or something you built that you are very proud of.